srtcore/list.h

```diff
@@ -71,12 +71,12 @@ class CSndLossList
 
    int insert(int32_t seqno1, int32_t seqno2);
 
-      /// Remove ALL the seq. no. that are not greater than the parameter.
+      /// Remove ALL sequence numbers that are less than or equal to the parameter.
```

Comment:
Sequence numbers are circular numbers - they are not less/greater, but earlier/later. Maybe:

Remove the given sequence number and all numbers that precede it.